scripts/asm-installer/tests/setup_longterm_cluster

```diff
@@ -9,71 +9,78 @@ SDIR="$(dirname "${SPATH}")"
 . "${SDIR}/common.sh"
 cd "${SDIR}"
 
-LT_CLUSTER_NAME="long-term-test-cluster"
-LT_CLUSTER_LOCATION="us-central1-c"
-LT_PROJECT_ID="asm-scriptaro-oss"
+EXPECTED_CLUSTER_NAMES="$1"
```

Comment:
I think you're making this too complicated. Just make an LT_CLUSTER_NAME and LT_ENVIRON_CLUSTER_NAME variable, put everything in this script into a function by wrapping it with `setup_cluster() {}`, and then put

```
setup_cluster "${LT_CLUSTER_NAME}"
setup_cluster "${LT_ENVIRON_CLUSTER_NAME}"
```

There's no need to start accepting input or do validation in a test setup script.

Reply:
I did that just to help our testing pipeline run a little faster and also it is easier to know from the cloudbuild file clusters to be set up.
But I am also okay with what you are suggesting here.